.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @chef-cookbooks/cookbook_engineering_team
+* sous-chefs/grafana
```

Comment:
You want a new team for this in terraform?

Reply:
Possibly, and might want  to name it keepalived lol

Reply:
Open a PR and I'll import the resource